LMChatUI_iOS/LMChatUI_iOS/Source/UI/PollWidget/CreatePollWidgets/LMChatCreatePollQuestionView.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  LMChatCreatePollQuestionView.swift
+//  LikeMindsFeedUI
+//
+//  Created by Pushpendra Singh on 17/07/24.
+//
+
+import UIKit
+
+public protocol LMChatCreatePollQuestionViewProtocol: AnyObject {
+    func onCrossButtonTapped(for id: Int)
+    func textValueChanged(for id: Int, newValue: String?)
+    func onAddNewOptionTapped()
+}
+
+open class LMChatCreatePollQuestionView: LMView {
+    // MARK: UI Elements
+    open private(set) lazy var containerView: LMView = {
+        let view = LMView().translatesAutoresizingMaskIntoConstraints()
+        return view
+    }()
+    
+    open private(set) lazy var answerLabel: LMLabel = {
+        let label = LMLabel().translatesAutoresizingMaskIntoConstraints()
+        label.text = "Answer options"
+        label.font = Appearance.shared.fonts.buttonFont2
+        label.textColor = Appearance.shared.colors.appTintColor
+        return label
+    }()
+    
+    open private(set) lazy var optionStack: LMTableView = {
+        let table = LMTableView().translatesAutoresizingMaskIntoConstraints()
+        table.dataSource = self
+        table.delegate = self
+        table.isScrollEnabled = false
+        table.showsVerticalScrollIndicator = false
+        table.showsHorizontalScrollIndicator = false
+        table.register(LMUIComponents.shared.createPollOptionCell)
+        table.estimatedRowHeight = cellSize
+        table.rowHeight = UITableView.automaticDimension
+        table.separatorStyle = .none
+        return table
+    }()
+    
+    open private(set) lazy var addOptionView: LMView = {
+        let view = LMView().translatesAutoresizingMaskIntoConstraints()
+        view.backgroundColor = .clear
+        view.isUserInteractionEnabled = true
+        return view
+    }()
+    
+    open private(set) lazy var addOptionImage: LMImageView = {
+        let image = LMImageView(image: Constants.shared.images.plusCircleIcon.withConfiguration(UIImage.SymbolConfiguration(font: Appearance.shared.fonts.buttonFont2)))
+        image.translatesAutoresizingMaskIntoConstraints = false
+        return image
+    }()
+    
+    open private(set) lazy var addOptionText: LMLabel = {
+        let label = LMLabel().translatesAutoresizingMaskIntoConstraints()
+        label.text = "Add an option..."
```

Comment:
Can we move the hardcoded strings to constant file?

Reply:
Done